// Copyright 2018 The Chromium Authors
// Use of this source code is governed by a BSD-style license that can be
// found in the LICENSE file.
//
// See platform2/media_perception/mojom/hotword_detection.mojom for relevant
// comments on data fields.

syntax = "proto2";

package mri;

enum HotwordType {
  HOTWORD_TYPE_UNKNOWN = 0;
  OK_GOOGLE = 1;
}

message Hotword {
  optional HotwordType type = 1;

  optional int64 start_timestamp_ms = 2;

  optional int64 end_timestamp_ms = 3;
}

message HotwordDetection {
  repeated Hotword hotwords = 1;
}
